alvr/server_openvr/cpp/alvr_server/HMD.cpp

```diff
@@ -367,3 +367,11 @@ void Hmd::GetProjectionRaw(vr::EVREye eye, float* left, float* right, float* top
 vr::DistortionCoordinates_t Hmd::ComputeDistortion(vr::EVREye, float u, float v) {
     return { { u, v }, { u, v }, { u, v } };
 }
+
+bool Hmd::ComputeInverseDistortion(
+    vr::HmdVector2_t* pResult, vr::EVREye eEye, uint32_t unChannel, float fU, float fV
+) {
+    // Fail and hope everything is fine, this assumes false == fail the documentation doesn't
+    // specify
```

Comment:
We don't have to program that defensively that we can even think about `failure = true`. Also we don't hope since the docs of the function specify that failing is fine. I.e. remove the comment.